### PR TITLE
DL-5515 play-frontend-govuk upgrade

### PR DESCRIPTION
**Pull request by jmountain-hmrc**
# DL-5515 play-frontend-govuk upgrade

**Frontend library upgrade*

* Upgraded ACM to use play-frontend-govuk

## Checklist

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [x]  I've added my code using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/uk/gov/hmrc/agentclientmandate/views/client/collectEmail.scala.html [hunk 1]
```diff
@@ -19,28 +19,32 @@
...
+@this(main_template: main_template, formHelper: FormWithCSRF, govukInput: GovukInput)
 
 @(service: String, addEmailForm: Form[ClientEmail], mode: Option[String], backLink: Option[String])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @main_template(title = Messages("client.collect-email.title"), delegatedService = Some(service)) {
   @pageHeadersAndError(backLink, "pre-heading", Messages("client.collect-email.preheader"), "header", Messages("client.collect-email.header"),
     Some(mandateErrorSummary(addEmailForm, "client.collect-email.error.general")))
 
-    <p id="lede">
+    <p class="govuk-body" id="govuk-body-l">
```
Comment (benryan1790) [inline app/uk/gov/hmrc/agentclientmandate/views/client/collectEmail.scala.html:32]: Change of id here, looks like the name of a class. Was this intentional?

File: app/uk/gov/hmrc/agentclientmandate/views/client/editEmail.scala.html [hunk 2]
```diff
@@ -18,34 +18,37 @@
...
 @(service: String, addEmailForm: Form[ClientEmail], backLink: Option[String])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @main_template(title = Messages("client.edit-email.title"), delegatedService = Some(service)) {
 @pageHeadersAndError(backLink, "pre-heading", Messages("client.edit-email.preheader", service), "header", Messages("client.edit-email.header"),
 Some(mandateErrorSummary(addEmailForm, "client.edit-email.error.general")))
 
-<div class="form-group">
-    <p id="lede">
+<div class="govuk-form-group">
+    <p class="govuk-body-l">
```
Comment (benryan1790) [inline app/uk/gov/hmrc/agentclientmandate/views/client/editEmail.scala.html:34]: Similar to previous comment, was this change of id to class intentional?

File: app/uk/gov/hmrc/agentclientmandate/views/client/collectEmail.scala.html (hunk 1 above)
Comment (jmountain-hmrc) [reply app/uk/gov/hmrc/agentclientmandate/views/client/collectEmail.scala.html:32]: It seemed wrong to me as per https://design-system.service.gov.uk/get-started/updating-your-code/ "lede" is used a class there, so I inferred it should be a class - and additionally converted to govuk-body-l in the process

File: app/uk/gov/hmrc/agentclientmandate/views/client/editEmail.scala.html (hunk 2 above)
Comment (jmountain-hmrc) [reply app/uk/gov/hmrc/agentclientmandate/views/client/editEmail.scala.html:34]: Yes, as above

**Review by benryan1790 — APPROVED** [review]

File: app/uk/gov/hmrc/agentclientmandate/views/client/collectEmail.scala.html (hunk 1 above)
Comment (benryan1790) [reply app/uk/gov/hmrc/agentclientmandate/views/client/collectEmail.scala.html:32]: So shouldn't you remove `class="govuk-body"` and change `id="govuk-body-l"` to `class="govuk-body-l"`?

Comment (jmountain-hmrc) [reply app/uk/gov/hmrc/agentclientmandate/views/client/collectEmail.scala.html:32]: Potentially, I can do that

**Review by benryan1790 — APPROVED** [review]

**Pull request merged**